IPython/terminal/interactiveshell.py

```diff
@@ -58,6 +56,26 @@ class _NoStyle(Style): pass
             Token.OutPromptNum: '#bb0000 bold',
 }
 
+_style_overrides_neutral = {
+            Token.Number: '#007700',
+            Token.Operator: 'noinherit',
+            Token.String: '#BB6622',
+            Token.Name.Function: '#2080D0',
+            Token.Name.Class: 'bold #2080D0',
+            Token.Name.Namespace: 'bold #2080D0',
+            Token.Prompt: '#009900',
+            Token.PromptNum: '#00ff00 bold',
+            Token.OutPrompt: '#990000',
+            Token.OutPromptNum: '#ff0000 bold'
+}
+
+_style_overrides_windows = {
+            Token.Prompt: '#ansidarkgreen',
+            Token.PromptNum: '#ansigreen bold',
+            Token.OutPrompt: '#ansidarkred',
+            Token.OutPromptNum: '#ansired bold'
+}
+
```

Comment:
THat seem more uniform, though that will conflict will already existing PR it would be better to do as separate changes. I'll see which PR is easier to apply first.